index.html

```diff
@@ -6,9 +6,11 @@
 
 <!-- This loops through the paginated posts -->
 {% for post in paginator.posts %}
+{% unless post.categories contains "slides" %}
```

Comment:
I don't think I understand this. Could you elaborate?

Reply:
Yeah this is not clear and the most confusing part about this PR. Since slides are posts they are present in the Liquid `paginator.posts` and `site.posts`. So that only the main event page shows up and can be navigated to I skip posts that are slides.

I don't know how to make this simpler so I will add a comment to the source.

I also found that I don't need to introduce a category for slides, and can instead key off the layout. I've updated the PR.

Reply:
👍